### DISCUSSION TITLE
Direct links to items

### DISCUSSION THREAD

**Discussion by mfominov** [dani-garcia/vaultwarden#2161] [Ideas]
Hi, first of all i want to say thx for this project it's really awesome.

This is not a bug, it's a kind of feature request.

It will be useful to create direct links to secrets for sharing with someone.

**Comment by BlackDex** [comment]
This is something for the clients, which we do not maintain our selfs.
Also, feature requests for Vaultwarden should actually go to https://github.com/dani-garcia/vaultwarden/discussions/categories/ideas

If you have feature requests for clients then i suggest to go to this page: https://community.bitwarden.com/c/feature-requests/5

**Reply by mfominov** [reply]
I'm talking about web ui.
Is there any solution to create link to specific secret?

**Reply by BlackDex** [reply]
The web ui is also a client maintained by Bitwarden.

There isn't a direct link option available, There are some search links which only work when the audit/event logging is working, but that isn't really the same and it will also popup some other windows not wanted.
But again, not working for Vaultwarden currently.

**Reply by mfominov** [reply]
Thx
